tasks/pipeline.py

```diff
@@ -243,11 +243,11 @@ def run(
 ):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What if instead we use the task's help to document the command? The advantage is that if we remove a param from the task, the linter will help us removing it from the documentation too

Could not make an inline suggestion, but would be something like
```python
@task(
	help={
		"here": "Run against current branch, default False",
		"use_release_entries": "use release entries, used with deploy=True, default False",
		"deploy": "trigger deployment at the end of the pipeline, default False",
		"deploy_installer": "trigger installer deployment to staging, default False",
		"all_builds": "run builds for all architectures, default True",
		"e2e_tests": "run all e2e tests, default True",
		"kmt_tests": "tun all kmt tests, default True"
	}
)
def run(
    ctx,
    git_ref="",
    here=False,
    use_release_entries=False,
    major_versions=None,
    repo_branch="dev",
    deploy=False,
    deploy_installer=False,
    all_builds=True,
    e2e_tests=True,
    kmt_tests=True,
    rc_build=False,
    rc_k8s_deployments=False,
):
```

Reply:
Thank you for your comment, my only goal was to sync the example string with the actual parameters, i don't want to make any functional changes.

Reply:
Would not be a functional change, but I understand it is a larger change than what you planned